emails/templates/emails/wrapped_email.html

```diff
@@ -12,110 +12,155 @@
           {{ original_html|safe }}
         </td>
       </tr>
     </table>
-    <table width="100%" align="center" bgcolor="#f9f9fa" style="background: #f9f9fa; padding-top: 12px; padding-right: 12px; padding-left: 12px; padding-bottom:12px; margin-top: 0px; margin-bottom: 30px; width: 100%;"">
+
+    <table width="100%" bgcolor="#3D3D3D" style="background: #3D3D3D; padding: 12px 25px; margin-top: 30px; margin-bottom: 30px; width: 96%; border-radius: 6px; max-width: 1200px;" align="center">
       <tr>
-        <td align="center" valign="top" width="100%" style="max-width:700px; padding-top: 12px; padding-bottom: 24px;">
-          <table align="center" width="100%" style="max-width: 700px; border-collapse: collapse; padding-top: 0px; padding-bottom: 0px;">
-            <tr>
-              <td align="center" width="100%" style="padding-top: 0px; padding-bottom: 8px; text-align: center">
-                <img width="160" src="{{ SITE_ORIGIN }}/static/images/email-images/relay-logo-emails.png" style="display: inline-block; margin-bottom: 0px; width: 160px;" alt="Relay Logo" />
-              </td>
-            </tr>
-            <tr>
-              <td align="center" style="line-height: 150%; padding-left: 20px; padding-right: 20px;">
-                <p style="display: inline-block; padding-top: 0; font-size: 13px; color: #363959; font-family: sans-serif; padding-left: 20px; padding-right: 20px; margin-top: 0; margin-bottom: 0;">
-                  <a href="{{ SITE_ORIGIN }}/accounts/profile?utm_source=emails&utm_medium=email&utm_campaign=alias-email" target="_blank" rel="noopener noreferrer" style="font-family: sans-serif; color: #20123a; text-decoration: underline; font-weight: bolder; font-size: 13px;">{% ftlmsg 'forwarded-email-footer-2' %}</a>
-                  {% if not has_premium and in_premium_country %}
-                    |
-                    <a href="{{ SITE_ORIGIN }}/premium?utm_source=emails&utm_medium=email&utm_campaign=alias-email&utm_content=footer" target="_blank" rel="noopener noreferrer" style="font-family: sans-serif; color: #20123a; text-decoration: underline; font-weight: bolder; font-size: 13px;">{% ftlmsg 'forwarded-email-footer-premium-banner' %}</a>
-                  {% endif %}
-                </p>
-              </td>
-            </tr>
+        <td align="center" valign="top" width="100%" style=" padding-top: 0px; padding-bottom: 0px;">
+          <table width="100%" style="max-width:1200px; text-align: center; border-collapse: collapse;">
+              <tr>
+                <td class="footer-block" width="50%" align="left">  
+                  <a class="container-link" href="{{ SITE_ORIGIN }}/accounts/profile"> 
+                    <img width="110" src="{{ SITE_ORIGIN }}/static/images/email-images/relay-logo-emails-dark-bg.png" style="margin: 0;" alt="relay logo"/>   
+                  </a>
+                </td>
+                <td class="footer-block" width="50%" align="right">  
```

Comment:
In the designs, there's also an "Upgrade to Relay Premium" link here if the user does not have Premium. (And I'm assuming that should link to `{{ SITE_ORIGIN }}/premium`.)